src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container stars--0">
```

Comment:
<img width="875" alt="image" src="https://user-images.githubusercontent.com/99403198/211149866-53c97920-2f15-4aca-8047-221a143204d4.png">

```suggestion
    <div class="stars stars--0">
```